### PR TITLE
Integrated boilerplate structure to Home files.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: quupe-RN/js/screens/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// import React, { Component } from 'react';
```
Comment (10thfloor) [inline quupe-RN/js/screens/Home/Home.js]: Please do not include commented code in your pull-requests.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: quupe-RN/js/screens/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+import PropTypes from 'prop-types';
```
Comment (siddharthparmar7) [inline quupe-RN/js/screens/Home/Home.js:4]: You are not using this module so take out this line.

File: quupe-RN/js/screens/Home/HomeContainer.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
```
Comment (siddharthparmar7) [inline quupe-RN/js/screens/Home/HomeContainer.js:2]: You are not using Text in your code. Remove it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes and add ESLint to your project if you have not so far. Also, add dependencies if you have any with this pull request. Thank you.

**Review by siddharthparmar7 — COMMENTED** [review]
Also, please don't forget to squash your commits.

**Pull request closed without merging**